File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.java
class Solution {
    public int[] solution(String my_string) {
        int[] answer = new int[52];
        for(int i = 0; i < my_string.length(); i++){
            char c = my_string.charAt(i);
            if(c - 'a' > -1 ){
                answer[c-97+26]++;
            }else {
                answer[c-65]++;
            }
            
        }
        return answer;
    }
}

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// 파라미터로 주어지는 문자열은 const로 주어집니다. 변경하려면 문자열을 복사해서 사용하세요.
int* solution(const char* my_string) {
    int len = strlen(my_string);
    int* answer = (int*)calloc(52, sizeof(int));
    
    for(int i = 0; i < len; i++){
        if(my_string[i] < 97){
            answer[my_string[i] - 65]++;
        }else {
            answer[my_string[i] - 97 + 26]++;
        }
    }
    
    return answer;
}